tkg/client/client.go

```diff
@@ -83,6 +83,7 @@ type InitRegionOptions struct {
 	CeipOptIn                    bool
 	UseExistingCluster           bool
 	IsInputFileClusterClassBased bool
+	GenerateOnly                 bool
```

Comment:
It seems to me DryRun is more straightforward.

Reply:
As other options structs have already used this name , I think we'd better to be consistent with other structs.